docs/workflows/reconnection-logic.md

```diff
@@ -0,0 +1,91 @@
+<%(TOC)%>
+
+# Handling disconnections
+
+Online games rely on all the clients and the server being able to connect and stay connected during the game. However, a client may disconnect due to an unstable connection or when the application is put into a background state by the OS. In this page, we will dive a bit deeper into the causes of a disconnect and discuss how you can handle it in your game.
+
+## Why do disconnects happen
+
+### Unstable connection 
+
+You need to ensure that [client-workers]({{urlRoot}}/reference/glossary#client-worker) are able to try to connect even when the connect might be more unstable. If the connection becomes too weak or unstable, there is a high risk of them disconnecting soon. It is important to include logic in your game that detects and handles disconnects and provides ways of reconnecting your clients.
```

Comment:
```suggestion
You need to ensure that [client-workers]({{urlRoot}}/reference/glossary#client-worker) are able to try to connect even when the connect might be more unstable. If the connection becomes too weak or unstable, there is a risk of them disconnecting. It is important to include logic in your game that detects and handles disconnects and provides ways of reconnecting your clients.
```

Reply:
> You need to ensure that [client-workers]({{urlRoot}}/reference/glossary#client-worker) are able to try to connect even when the connect might be more unstable.

Why is it important to be able to connect when your connection is unstable? There's not a lot a customer could do about this

Reply:
I guess the wording here is kinda off. it is important to ensure that client disconnects won't lead the client in a bad state.